src/components/form-helpers/form-helpers.js

```diff
@@ -267,22 +267,25 @@ const _formatFieldValueAsInternationalNumber = (iti, fieldName, setFieldValue) =
 
 /**
  * Phone Field Item Component
- * @param { Object } props - props
- * @param { import('react-intl').InjectedIntl } props.intl - intl
- * @param { import('formik').FormikProps<Values> } props.formik - formik
- * @param { string } props.className - className
- * @param { string } props.fieldName - fieldName
- * @param { string } props.label - label
- * @param { string } props.placeholder - placeholder
- * @param { React.MutableRefObject<import('intl-tel-input').Plugin> } props.intlTelInputRef - intlTelInputRef
- */
+
+  * @param { Object } props - props
```

Comment:
```suggestion
  * @param { Object } props - props
```

Reply:
or maybe

```suggestion
  *
  * @param { Object } props - props
```

Please, check JSDoc documentation to use the right format